analytical/templatetags/piwik.py

```diff
@@ -45,6 +55,13 @@ def piwik(parser, token):
     Renders Javascript code to track page visits.  You must supply
     your Piwik domain (plus optional URI path), and tracked site ID
     in the ``PIWIK_DOMAIN_PATH`` and the ``PIWIK_SITE_ID`` setting.
+
+    Custom variables can be passed in the ``piwik_vars`` context
+    variable. It is an iterable of custom variables as tuples like:
+    ``(index, name, value[, scope])`` where scope may be ``'page'``
+    (default) or ``'visit'``. Index should be an integer and the
+    other parameters should be strings.
+    See the [Piwik documentation](http://developer.piwik.org/guides/tracking-javascript-guide#custom-variables)
```

Comment:
The link should be reST syntax, you've made it in Markdown.

Use
``` reST
    See the `Piwik documentation <http://developer.piwik.org/guides/tracking-javascript-guide#custom-variables>`_
```
or (preferred)
``` reST
    See the `Piwik documentation`_

    .. _Piwik documentation: http://developer.piwik.org/guides/tracking-javascript-guide#custom-variables
```